### DISCUSSION TITLE
Cache in Sktime (or Rocket)

### DISCUSSION THREAD

**Discussion by haskarb** [alan-turing-institute/sktime#1530] [Q&A] [answered]
I have observed peculiar phenomena while running Rocket from Sktime. When I run Rocket on a new machine that doesn't have sktime installed previously, the computation time I get is around 12 minutes for EigenWorms (MTSC dataset); however, after few more runs, the time is reduced to few minutes? Is there any cache storage happening?

**Comment by fstinner** [comment, marked as answer]
Sktime uses numba and in Rocket the cache of numba is on. I assume that a cache is used.

**Comment by haskarb** [comment]
Interesting! Any idea how we can turn that off? I suspect I might need to go to the source code of Rocket.

**Comment by fstinner** [comment]
In my opinion, it is recommended to use numba's cache. This saves only the compiled part of the program.
